README.md

```diff
@@ -31,11 +31,9 @@ const server = node.createServer()
 
 server.on('connection', function (socket) {
   // socket is E2E encrypted between you and the other peer
-  // pipe it somewhere like any duplex stream
-
   console.log('Remote public key', socket.remotePublicKey)
-  console.log('Local public key', socket.publicKey) // same as keyPair.publicKey
```

Comment:
If there is no need to emphasize `socket.publicKey` I would say we can remove it as it's not very common to use it (at least not when first trying it out)